### ISSUE TITLE
Localisation

### ISSUE THREAD

**Issue by genna87** [squarecat/pokemongo-data#1]
Do you plan to make a multilingual version?

**Comment by Jivings** [comment]
@genna87 only if someone wanted to contribute translations :)

**Comment by genna87** [comment]
How can I help?

**Comment by Jivings** [comment]
@genna87 If you take a look at [`master.json`](https://github.com/squarecat/pokemongo-data/blob/master/assets/master.json). This is the decoded file that comes from the Pokemon Go app. 

In that file you'll see strings with underscores such as `POKEMON_TYPE_GRASS` or `V0013_MOVE_WRAP` for each item. Ideally these should be replaced at runtime with a proper localised form of the name. For this we need a lookup table so that we know what word to replace them with. For instance;

``` js
var lang = {
  types: {
    POKEMON_TYPE_GRASS: {
      en: "grass",
      fr: "plante",
      ...
    },
    ...
  },
  moves: {
    V0013_MOVE_WRAP: {
      en: "wrap",
      fr: "Ligotage"
    },
    ...
  }
}
```

Usefully, Bulbapedia has a list of [moves](http://bulbapedia.bulbagarden.net/wiki/List_of_moves_in_other_languages) and [types](http://www.pokepedia.fr/index.php/Type) in other languages, which would be a great reference.

I hope this makes sense! It would be a great step forward :)

**Comment by genna87** [comment]
I'll try creating the table starting from there

It would need some time, tho

**Comment by Jivings** [comment]
@genna87 I'm going to create the English version tonight, generated from the `master.json`, that can then be improved by others.

**Comment by genna87** [comment]
Great

**Comment by genna87** [comment]
Other improvement suggestions:
- Link to Pokémon card on images of evolutions
- Close card clicking outside
- Search bar
- Items

PS
Could you explain me the meaning of the green and blue bars of the attacks?

THANKS!

**Comment by Jivings** [comment]
Movedex is coming next :)

The green bars mean that the move **replenishes** stamina, the blue one uses it up. 

See, when you're battling and you use the fast move it charges up the stamina bar, and when you use the special move it consumes it. I will document these things in the movedex!

**Comment by genna87** [comment]
I've isolated and grouped all the translatable strings in the [master.json](https://github.com/squarecat/pokemongo-data/blob/master/assets/master.json) file:
[pkmn-go-translations-id.txt](https://github.com/squarecat/pokemongo-data/files/373934/pkmn-go-translations-id.txt)

Let me know which groups you think should be translated

**Comment by Jivings** [comment]
That's awesome! Nice one. 

I also made a file with mapping some of the values that I am using to English:
https://github.com/squarecat/pokemongo-data/blob/master/assets/lang.json

Lets build on this file with other languages :)

**Comment by genna87** [comment]
Here a first draft for English and Italian:
[lang.json.txt](https://github.com/squarecat/pokemongo-data/files/376360/lang.json.txt)

**Comment by genna87** [comment]
Where can I find the other strings to be translated?

EG:
- "Basic Attacks"
- "Basic attacks are fast and replenish energy."
- ...

**Comment by Jivings** [comment]
That file looks great! I'll make everything use it sometime this weekend hopefully. 

The other strings do not have a file yet, but it will be in the same directory and will be a similar format. eg.

```
{
  "movedex": {
    "title": {
        "en": "basic attacks"
    },
    "description": {
      "en": "Basic attacks are fast and replenish energy."
    }
}
```

 Thanks for your help!

**Comment by genna87** [comment]
Any news?

How can I help?

**Comment by Jivings** [comment]
Yes! I have made significant progress. You'll now see many lang files [here](https://github.com/squarecat/pokemongo-data/tree/master/assets/lang) for each asset type. I have included your translations where necessary. 

There are translations missing for `filters`, `item` descriptions, `movedex` page descriptions, and `pokedex` page description.

I will work on adding a option to set language at some point this week so we will be able to see how it looks and what is missing.

**Comment by genna87** [comment]
Great

Looking forward for any update
